data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/MutateMapper.java

```diff
@@ -64,54 +79,94 @@ public RenameCopyConfig(String fromKey, String toKey) {
             } else if(Objects.equals(name, "uppercase")) {
                 uppercases.addAll(((ArrayList<String>) attr.getAttributeValue().getValue()).stream()
                         .map(NestedSyntaxConverter::convertNestedSyntaxToJsonPointer).collect(Collectors.toList()));
+            } else if(Objects.equals(name, "lowercase")) {
+                lowercases.addAll(((ArrayList<String>) attr.getAttributeValue().getValue()).stream()
+                        .map(NestedSyntaxConverter::convertNestedSyntaxToJsonPointer).collect(Collectors.toList()));
+            } else if(Objects.equals(name, "trim")) {
+                trims.addAll(((ArrayList<String>) attr.getAttributeValue().getValue()).stream()
+                        .map(NestedSyntaxConverter::convertNestedSyntaxToJsonPointer).collect(Collectors.toList()));
+            } else if(Objects.equals(name, "gsub")) {
+                final List<String> array = ((ArrayList<String>) attr.getAttributeValue().getValue()).stream()
+                        .map(NestedSyntaxConverter::convertNestedSyntaxToJsonPointer).collect(Collectors.toList());
```

Comment:
Let's rename this variable from `array` to something like `gsubSettings`

Reply:
This entire code block is about to be blown up and refactored. This is just a stop-gap